src/libraries/System.Private.CoreLib/src/System/Text/Ascii.Equality.cs

```diff
@@ -455,9 +477,12 @@ private interface ILoader<TLeft, TRight>
             static abstract nuint Count128 { get; }
             static abstract nuint Count256 { get; }
             static abstract nuint Count512 { get; }
+            static abstract bool Use64Bit { get; }
```

Comment:
If the `PlainLoader` take the HW into account (see comment) above, this could become
```suggestion
            static virtual bool Use64Bit => IntPtr.Size == 8 && Vector128.IsHardwareAccelerated;
```
and remove the methods from the struct implementations.

Maybe a better name also? As `Use64Bit` doesn't take the HW into account.
Naming is hard, but something like `Is64BitAndHardwareAccelerated`?